src/Structure/ClassStructureTrait.php

```diff
@@ -129,6 +135,16 @@ public function jsonSerialize()
             }
         }
 
+        if ($this->extendedPropertySerialization) {
```

Comment:
Let's remove this check (with associated field and setter) and unconditionally export additional and pattern properties, so that import-export round trip is lossless.